overlord/servicestate/quota_control_test.go

```diff
@@ -273,7 +288,60 @@ func (s *quotaControlSuite) TestCreateQuotaPreseeding(c *C) {
 	})
 }
 
-func (s *quotaControlSuite) TestCreateQuota(c *C) {
+func (s *quotaControlSuite) TestCreateUpdateRemoveQuotaHappy(c *C) {
```

Comment:
nitpick: maybe this one should go at the end after all the Quota* tests?

Reply:
It will be at the end of the file in my followup PR where I move all these unit tests that do handler things to the quota_handlers_test.go file